packages/office-ui-fabric-react/src/utilities/decorators/withResponsiveMode.tsx

```diff
@@ -75,8 +75,12 @@ export function withResponsiveMode<P extends { responsiveMode?: ResponsiveMode }
       let win = getWindow();
 
       if (typeof win !== 'undefined') {
-        while (win.innerWidth > RESPONSIVE_MAX_CONSTRAINT[responsiveMode]) {
-          responsiveMode++;
+        try {
+          while (win.innerWidth > RESPONSIVE_MAX_CONSTRAINT[responsiveMode]) {
+            responsiveMode++;
+          }
+        } catch (e) {
+          responsiveMode = undefined;
```

Comment:
It seems like you should fall back to  `_defaultMode`. Returning undefined is unexpected, given it doesn't map to any enum value.